src/main/java/frc/robot/motion_profiling/FollowPath.java

```diff
@@ -0,0 +1,141 @@
+package frc.robot.motion_profiling;
```

Comment:
I thought you said that packages are written like this: `frc.robot.motionprofiling` each folder with one word